sessions/recap-project-5/example-implementation/pages/art-pieces/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import ArtPiecesPreview from "../../components/ArtPiecesPreview";
+
+const ImageGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(1fr);
```

Comment:
repeat without a number does not work. Also 1fr of 100% does not work as well (i think?!).

You probably mean something like this:
```suggestion
  grid-auto-rows: 200px;
```